pkg/rtc/mediatracksubscriptions.go

```diff
@@ -98,15 +101,25 @@ func (t *MediaTrackSubscriptions) AddSubscriber(sub types.LocalParticipant, wr *
 	for _, c := range codecs {
 		c.RTCPFeedback = rtcpFeedback
 	}
-	downTrack, err := sfu.NewDownTrack(
-		codecs,
-		wr,
-		sub.GetBufferFactory(),
-		subscriberID,
-		t.params.ReceiverConfig.PacketBufferSize,
-		sub.GetPacer(),
-		LoggerWithTrack(sub.GetLogger(), trackID, t.params.IsRelayed),
-	)
+
+	streamID := wr.StreamID()
+	if sub.ProtocolVersion().SupportSyncStreamID() && t.params.MediaTrack.Stream() != "" {
+		streamID = PackSyncStreamID(t.params.MediaTrack.PublisherID(), t.params.MediaTrack.Stream())
+	}
+	downTrack, err := sfu.NewDownTrack(sfu.DowntrackParams{
+		Codecs:        codecs,
+		Receiver:      wr,
+		BufferFactory: sub.GetBufferFactory(),
+		SubID:         subscriberID,
+		StreamID:      streamID,
+		MaxTrack:      t.params.ReceiverConfig.PacketBufferSize,
+		PlayoutDelayLimit: rtpextension.PlayOutDelay{
+			Min: uint16(t.params.VideoConfig.PlayoutDelay.Min / 10),
+			Max: uint16(t.params.VideoConfig.PlayoutDelay.Max / 10),
```

Comment:
why the divide by 10? Maybe add a comment? Is it because play out delay extension uses a factor of 1/10? If so, would be good to pass the config and apply the change where it is used.